components/Navbar.jsx

```diff
@@ -0,0 +1,79 @@
+import { Menu } from "@headlessui/react";
+import Image from "next/image";
+import Link from "next/link";
+import React, { useState } from "react";
+import Dropdown from "./Dropdown";
+import Fade from "./Fade";
+import { FaBars } from "react-icons/fa";
+import MobileNavbarDrawer from "./MobileNavbarDrawer";
+
+const textClasses = "text-xl font-semibold text-[#FF4D6F]";
+const optionClasses = "text-xl font-semibold text-white p-2";
+const optionsClasses = "absolute right-0 top-12 w-48 rounded-b-lg bg-[#FF4D6F] p-2";
+
+const DropdownOptions = ({ options }) => {
+  return (
+    <Menu.Items as="div" className={`absolute right-0 mt-2 origin-top-right divide-y ${optionsClasses}`}>
+      {options.map((option) => {
+        const link = option.toLowerCase().split(" ").join("-");
+        return (
+          <Menu.Item key={option} className={`flex justify-center items-center ${optionClasses}`}>
+            <a href={`/${link}</Menu.Item>`}>{option}</a>
+          </Menu.Item>
+        );
+      })}
+    </Menu.Items>
+  );
+};
```

Comment:
Would this be better suited in the `Dropdown.jsx` file?

Reply:
The navbar dropdown is quite different compared to other dropdowns we might have later. For example, it sits specifically on the bottom of the navbar. I was thinking when we build a more generic dropdown in another ticket, we can just accept a `prop` for options and default the `children` to render internally in the `Dropdown`

Reply:
Sounds good! In that case, can we name dropdown a bit more specifically? Like NavbarDropdown?

Reply:
Rename `DropdownOptions` to be `NavbarDropdownOptions` and then the `Dropdown` component we will re-use and add generic options to later.